### PR TITLE
Creating sample java application using jwt token

### LINKED ISSUE #34: [feature]: Creating Sample Java Application using JWT Token
Knowing how time based tokens like jwt and sessions work . 
And creating sample java application using jwt token authentication can be a good first issue!

### PR DESCRIPTION
**Pull request by reem-atalah**
# Pull Request Template

## Description
Creating sample java application using JWT token authentication and tested using Keploy.

Fixes #34 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Using the latest jar and putting the environment variables as used in samples-java

## Additional Context (Please include any Screenshots/gifs if relevant)
Capturing requests using Keploy record.
capturing_record_and_testing

First API: login and receive the token.
login

Second API: put token in authorization to access any other APIs.
success_hello

Record requests with Keploy
capturing_record

Test with Keploy
success_tests

...

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: samples-java-jwt/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    `</dependency>`
+
+or to *build.gradle*:
+
+    implementation 'io.keploy:keploy-sdk:N.N.N' (eg: 1.2.8)
+
+## Usage
+- Refer [this](https://github.com/keploy/java-sdk#usage).
+
+## Setup Authintication App
```
Comment (gouravkrosx) [inline samples-java-jwt/README.md:86]: correct the spelling of `Authentication`.

**Comment by gouravkrosx** [comment]
And please also re-structure this repo if possible, like whatever code is in this repo is for samples-java-sql application. Since we are adding more samples application hence project structure should look like.
### Samples-Java
- samples-java-sql
- samples-java-jwt
- samples-java-someOtherDependency
- README.md ( To mention all above sample applications)

You can take reference  from [this](https://github.com/keploy/samples-go).

**Comment by reem-atalah** [comment]
@gouravkrosx I have corrected the spelling and generalizes samples-java, you can check it now

**Comment by gouravkrosx** [comment]
Please review why your build is failing and
I) Remove the **Quickstart** `Gitpod` section from the Readme file.
2) Sample application links in the outer readme file are giving `404`.

**Pull request closed without merging**